src/registrar/templates/admin/change_list.html

```diff
@@ -24,3 +24,11 @@ <h2>
         {% endif %}
     </h2>    
 {% endblock %}
+
+{% block object-tools %}
+    <p class="object-tools">
```

Comment:
I think this approach works fine, but I'd make a note somewhere in the description or comments before closing as to the specifics of why we chose to do this. For our purposes I highly doubt we'd need to iterate through a list but it would be good to document this in the event we do